docs/components/modeler/web-modeler/save-and-deploy.md

```diff
@@ -0,0 +1,15 @@
+---
+id: save-and-deploy
+title: Save and deploy your diagram
+description: "If a diagram is changed and autosaved, it has no effect on your cluster."
+---
+
+Web Modeler will autosave all your changes on a diagram, moreover the changes will also be visible in real-time to any collaborators opening the same diagram.
+
+When you change a diagram and it is autosaved, that has no effect on your cluster(s).
+
+When you deploy the diagram, it becomes available on the selected cluster and new instances can start.
+
+To deploy, click **Deploy Diagram**:
```

Comment:
There is a mismatch between the image where sentence case is used and title case used in this sentence.